### PR TITLE
Feature - Enable wallets by default

### PR DESCRIPTION
**Pull request by dannypaz**
This PR removes the `noseedbackup` flag from LND as the default. This flag had previously allowed LND to initialize a wallet on behalf of the user that was not recoverable OR password protected.

By removing this flag, the user will now have to create a wallet on LND engine startup and each subsequent restart of the engine, will lock the lnd engine requiring a password.

In previous PRs we have supplied a `createWallet` and `unlockWallet` API which is used for both of these actions.

Additionally, this PR looks for a `NO_SEED_BACKUP` environment variable that enables the `--noseedbackup` flag for LND *ONLY* if the engine is set to something other than MainNet (real money)

### CODE REVIEW

**Comment by dannypaz** [comment]
[[EPIC] Remove `--no-seedbackup` configuration](https://trello.com/c/1VoOmUsh/853-epic-remove-no-seedbackup-configuration)

File: docker/lnd/start-lnd-btc.sh [hunk 1]
```diff
@@ -51,4 +47,13 @@ if [[ "$NODE" == "bitcoind" ]]; then
     PARAMS="$PARAMS --bitcoind.zmqpubrawtx=$ZMQPUBRAWTX"
 fi
 
+# We want to make it easy for devs to test functionality of the engine which could
+# potentially involve the constant restarting of a particular daemon. It can get
+# annoying to continually have to unlock/create wallets so we will enable noseedbackup
+# ONLY if the network is outside of mainnet AND dev is set to true
```
Comment (treygriffith) [inline docker/lnd/start-lnd-btc.sh]: your comment mentions that this only happens if dev is set to true, but the conditional doesn't check that. Which one is wrong?

Comment (dannypaz) [reply docker/lnd/start-lnd-btc.sh]: the comment is wrong. I changed the name from DEV to `NO_SEED_BACKUP`

**Comment by dannypaz** [comment]
Closing this PR. We can reopen when sparkswap is ready to fully lock down the engine's

**Pull request closed without merging**